modules/planning/lattice/trajectory_generation/trajectory1d_generator.cc

```diff
@@ -28,14 +28,43 @@
 #include "modules/common/log.h"
 #include "modules/planning/common/planning_gflags.h"
 #include "modules/planning/lattice/trajectory1d/constant_deceleration_trajectory1d.h"
-#include "modules/planning/lattice/trajectory1d/standing_still_trajectory1d.h"
 #include "modules/planning/lattice/trajectory1d/lattice_trajectory1d.h"
+#include "modules/planning/lattice/trajectory1d/standing_still_trajectory1d.h"
 #include "modules/planning/math/curve1d/quartic_polynomial_curve1d.h"
-#include "modules/planning/math/curve1d/quintic_polynomial_curve1d.h"
 
 namespace apollo {
 namespace planning {
 
+namespace {
+
+// A common function for trajectory bundles generation with 
+// a given initial and  end conditions.
+typedef std::array<double, 3> Init_Condition_T;
+typedef std::vector<std::pair<std::array<double, 3>, double>> End_Conditions_T;
+void GenerateTrajectoryBundlesWithCondition(
+    const Init_Condition_T& init_condition,
```

Comment:
Can you please rename the type name using Google coding style, maybe InitCondition, EndConditions

Reply:
Sure, no problem.

Reply:
OK. I'll modify it and submit again. Thank you.